"make" fails, unable to build

```
zimm@ZIMM_N85S:~/etherboy-core/ebgopath/src/github.com/loomnetwork/etherboy-core$ make
go build github.com/gogo/protobuf/protoc-gen-gogo
protoc --plugin=./protoc-gen-gogo -Ivendor -I/home/zimm/etherboy-core/ebgopath/src -I/usr/local/include --gogo_out=/home/zimm/etherboy-core/ebgopath/src \
github.com/loomnetwork/etherboy-core/txmsg/txmsg.proto
vendor: warning: directory does not exist.
protoc --plugin=./protoc-gen-gogo -Ivendor -I/home/zimm/etherboy-core/ebgopath/src -I/usr/local/include --gogo_out=/home/zimm/etherboy-core/ebgopath/src \
github.com/loomnetwork/etherboy-core/testdata/test.proto
vendor: warning: directory does not exist.
mkdir -p run/cmds
go build -o run/cmds/etherboycli tools/cli/etherboycli/etherboycli.go
# command-line-arguments
tools/cli/etherboycli/etherboycli.go:107:36: not enough arguments in call to contract.StaticCall
        have (string, *txmsg.StateQueryParams, *txmsg.StateQueryResult)
        want (string, proto.Message, loom.Address, interface {})
Makefile:19: recipe for target 'etherboy-cli' failed
make: *** [etherboy-cli] Error 2
```
OS: Ubuntu

--- a/tools/cli/etherboycli/etherboycli.go
+++ b/tools/cli/etherboycli/etherboycli.go
@@ -100,11 +100,23 @@ func main() {
 		Use:   "get",
 		Short: "get state",
 		RunE: func(cmd *cobra.Command, args []string) error {
+			privKey, err := getPrivKey(privFile)
+			if err != nil {
+				log.Fatal(err)
+			}
+
 			var result txmsg.StateQueryResult
 			params := &txmsg.StateQueryParams{
 				Owner: user,
 			}
-			if _, err := contract.StaticCall("GetState", params, &result); err != nil {
+
+			signer := auth.NewEd25519Signer(privKey)
+
+			callerAddr := loom.Address{
+				ChainID: rpcClient.GetChainID(),
+				Local:   loom.LocalAddressFromPublicKey(signer.PublicKey()),
+			}
+			if _, err := contract.StaticCall("GetState", params, callerAddr, &result); err != nil {
 				return err
 			}
 			fmt.Println(string(result.State))

